unstructured_ingest/v2/processes/connectors/redisdb.py

```diff
@@ -0,0 +1,209 @@
+import json
+import shutil
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import TYPE_CHECKING, Any, Optional
+
+from pydantic import Field, Secret
+
+from unstructured_ingest.error import DestinationConnectionError
+from unstructured_ingest.utils.data_prep import batch_generator
+from unstructured_ingest.utils.dep_check import requires_dependencies
+from unstructured_ingest.v2.interfaces import (
+    AccessConfig,
+    ConnectionConfig,
+    FileData,
+    Uploader,
+    UploaderConfig,
+    UploadStager,
+    UploadStagerConfig,
+)
+from unstructured_ingest.v2.logger import logger
+from unstructured_ingest.v2.processes.connector_registry import DestinationRegistryEntry
+
+if TYPE_CHECKING:
+    from redis.asyncio import Redis
+
+CONNECTOR_TYPE = "redis"
+SERVER_API_VERSION = "1"
+
+
+class RedisAccessConfig(AccessConfig):
```

Comment:
I thought we were creating a redis connector for each auth type supported?

Reply:
Well, of the platforms I have tested, there's actually just one auth type for now: password.
For Redis on Azure, it's either password (access key) or Microsoft Entra ID (this is in another ticket to be worked in the future).
For Redis on AWS ElastiCache, it's password.
For Upstash Redis, it's password.
For local docker container, it's password.
And it applies to Redis OpenSource, Enterprise, Cloud.
Do I miss anything here?